app/src/main/java/org/oppia/android/app/resumelesson/ResumeLessonActivityPresenter.kt

```diff
@@ -53,11 +93,44 @@ class ResumeLessonActivityPresenter @Inject constructor(
     }
   }
 
+  private fun retrieveReadingTextSize(): LiveData<ReadingTextSize> {
+    return Transformations.map(
+      profileManagementController.getProfile(profileId).toLiveData(),
+      ::processReadingTextSizeResult
+    )
+  }
+
+  private fun processReadingTextSizeResult(
+    readingTextSizeResult: AsyncResult<Profile>
+  ): ReadingTextSize {
+    return when (readingTextSizeResult) {
+      is AsyncResult.Failure -> {
+
+        Profile.getDefaultInstance()
```

Comment:
OKay.

Reply:
Look's like I commented under the pending branch instead of the failure branch which I intended. While a result is pending, we typically log with .d() because it is a debug not error meesage.